pyrk/neutronics.py

```diff
@@ -67,7 +71,7 @@ def init_rho_ext(self, rho_ext):
             rho_ext = ReactivityInsertion(self._timer)
         return rho_ext
 
-    def dpdt(self, t_idx, components, power, zetas):
+    def dpdt(self, t_idx, t_idx_feedback, components, power, zetas):
```

Comment:
t_idx_feedback and its definition should be added to the docstring here as you've successfully done in  a couple of other places.